src/cli.py

```diff
@@ -0,0 +1,59 @@
+"""Command line interface for the ingestion behavior."""
+import os
+import click
+from ingest import ingest_from_query
+
+@click.group()
+def cli():
+    pass
+
+
+@click.command()
+@click.option("--local_path", help="Parent directory for repository.")
+@click.option("--sub_path", help="The sub-directory of the local repository.")
+@click.option("--branch", default="main", help="The branch to use.")
+@click.option("--user_name", help="The user name of the repository owner.")
+@click.option("--repo_name", help="The logical name of the repository.")
+@click.option("--slug", help="A slug for the repo.")
+@click.option("--output", 
+              default="output",
+              help="The directory containing the output artifacts.")
+def ingest(local_path, sub_path, branch, user_name, repo_name, output, slug):
+    """Ingest a repository to create LLM-friendly ingestible data."""
+    _do_ingest(local_path, sub_path, branch, user_name, repo_name, output, slug)
+
+def _do_ingest(local_path, sub_path, branch, user_name, repo_name, output, slug):
```

Comment:
Why the need of a _do_ingest instead of doing this logic in ingest directly?

Reply:
Ah, I was refactoring and then decided against the other command I was going to add.  Didn't revert the refactored method.  It was going to be re-used.